### PR TITLE
Update default-crawled-file-name-extensions-and-parsed-file-types.md

### PR DESCRIPTION
**Pull request by petey121175**
Please work with Preethi Vemu <preethi.vemu@microsoft.com> and John Berkeley <johnber@microsoft.com> for final review.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/technical-reference/default-crawled-file-name-extensions-and-parsed-file-types.md [hunk 1]
```diff
@@ -32,12 +32,12 @@ By default, SharePoint Server satisfies these requirements for many file types.
 The following table shows all the file formats that SharePoint Server has built-in format handlers for. The table shows one or several format ID and file name extensions for each file format. By default SharePoint Server is enabled to parse files that have these file formats and file name extensions. For each file name extension the table also indicates whether the Manage File Types page by default includes the file name extension.
   
 > [!NOTE]
->  SharePoint Online supports the same file name extensions as in this table. In addition, SharePoint Online also supports the following: .one .xlt .xlc .xlb 
+>  SharePoint Online supports the same file name extensions as in the table, except for .eml or .mht files. In addition, SharePoint Online also supports the following: .one .xlt .xlc .xlb
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/technical-reference/default-crawled-file-name-extensions-and-parsed-file-types.md]:
```suggestion
>  SharePoint Online supports the same file name extensions as in the table, except for .eml and .mht files. In addition, SharePoint Online also supports the following: .one .xlt, .xlc, and .xlb.
```

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**